packages/article-feed-persoo/src/Resources/views/feed.xml.twig

```diff
@@ -0,0 +1,20 @@
+{% block begin %}
+<articles>
```

Comment:
isn't the XML declaration missing? (`<?xml version="1.0" encoding="utf-8"?>`)

Reply:
indeed, added in d73a25b5f1. I have also added it for the category feed in a separate commit (96dfbe142b). I see we also use an RSS version tag in some feeds but I do not feel like we need it here